pkgs/games/anki/default.nix

```diff
@@ -42,14 +42,10 @@ let
     fetchSubmodules = true;
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "linkcheck-0.4.1" = "sha256-S93J1cDzMlzDjcvz/WABmv8CEC6x78E+f7nzhsN7NkE=";
-      "percent-encoding-iri-2.2.0" = "sha256-kCBeS1PNExyJd4jWfDfctxq6iTdAq69jtxFQgCCQ8kQ=";
-    };
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    hash = "sha256-4V75+jS250XfUH6B4VBxtL2t308nyKzhDoq86kq6rp4=";
   };
```

Comment:
my understanding from looking at the docs last week was that this could be written as follow:
```
useFetchCargoVendor = true;
cargoHash = "sha256-4V75+jS250XfUH6B4VBxtL2t308nyKzhDoq86kq6rp4";
```

I don't have a hard preference, either get rid of the lock file which is a pain to update, if someone knows if there's a canonical/preferred way please chime in.

Reply:
I suppose this has the advantage that you don't need to copy the hash in sync-server, you can just reuse cargoDeps.

Reply:
Is there something that'd make cargoHash impossible to inherited just like cagoDeps?

But anyway, as said above, I'm not picky - this is just curiosity asking.

Reply:
Yes, that would also work.